wiki/Roll-Formulas.md

```diff
@@ -79,201 +71,163 @@
 
-`@attributes.init.bonus` - Any extra arbitrary bonus (active effects or the initiative config window)
+`@attributes.hp.temp` - Temporary hit points.
 
-`@attributes.init.total` - Final initiative modifier
+`@attributes.hp.tempmax` - Temporary changes to max hit points.
 
-#### Movement
+`@attributes.hp.pct` - Percentage hit points (current over maximum).
 
-`@attributes.movement.burrow` - 
+#### Initiative
 
-`@attributes.movement.climb` - 
+`@attributes.init.mod` - Actor's base initiative modifier derived from its Dexterity.
 
-`@attributes.movement.fly` - 
+`@attributes.init.prof` - Initiative [proficiency](Roll-Formulas#proficiency) details.
 
-`@attributes.movement.swim` - 
+`@attributes.init.total` - Final initiative modifier.
 
-`@attributes.movement.walk` - 
+`@attributes.init.roll.min` and `.max` - The minimum and maximum values of the d20 for initiative rolls.
 
-`@attributes.movement.units` - 
+#### Movement
 
-`@attributes.movement.hover` - 
+`@attributes.movement.burrow` - The actor's burrowing speed.
 
+`@attributes.movement.climb` - The actor's climbing speed.
 
-#### Senses
+`@attributes.movement.fly` - The actor's flying speed.
 
-`@attributes.senses.blindsight` - 
+`@attributes.movement.swim` - The actor's swimming speed.
 
-`@attributes.senses.darkvision` - 
+`@attributes.movement.walk` - The actor's walking speed.
 
-`@attributes.senses.tremorsense` - 
+#### Senses
 
-`@attributes.senses.truesight` - 
+`@attributes.senses.blindsight` - The actor's blindsight range.
 
-`@attributes.senses.units` - 
+`@attributes.senses.darkvision` - The actor's darkvision range.
 
-`@attributes.senses.special` - 
+`@attributes.senses.tremorsense` - The actor's tremorsense range.
 
+`@attributes.senses.truesight` - The actor's truesight range.
 
 #### Other Attributes
 
-`@attributes.exhaustion` - Current exhaustion level
-
-`@attributes.hd` - Currently available hit dice
-
-`@attributes.inspiration` - Whether the actor has inspiration
-
 `@attributes.prof` - Base, numerical proficiency value (does not reflect options like Proficiency Dice)
 
-`@attributes.spellcasting` - Spellcasting ability (three-letter code, not the modifier)
-
 `@attributes.spelldc` - Spell save DC based on the selected spellcasting ability
 
 `@attributes.spellmod` - Base ability modifier for the actor's selected spellcasting ability
 
-### Bonuses
-
-#### Ability
-
-`@bonuses.abilities.check` - Global ability check bonuses (added to base ability checks and skills checks)
-
-`@bonuses.abilities.save` - Global ability save bonuses
-
-`@bonuses.abilities.skill` - Global skill check bonuses
-
-#### Attacks
-
-`@bonuses.msak` - Melee spell attack
-
-`@bonuses.mwak` - Melee weapon attack
-
-`@bonuses.rsak` - Ranged spell attack
-
-`@bonuses.rwak` - Ranged weapon attack
-
-***Note:** Replace the `*` in the following formulas with the four-letter code from above.*
-
-`@bonuses.*.attack` - Global bonus to attack rolls
-
-`@bonuses.*.damage` - Global bonus to damage rolls
```

Comment:
Why are we removing these? Did they get moved somewhere else?

Reply:
Same as before, they can't be used as roll data. But they are in the AE wiki.

Reply:
Hmm, I see. Well it's probably fine so long as they're in the AE wiki, but some of them, like `@bonuses.mwak` certainly could be used in roll data so long as they don't also contain roll data references themselves. I suppose I don't really know when you would want to do that though.

Reply:
I think excluding properties that can lead to issues in that way is a good rule of thumb to use (else we could or should technically include `@name` if you name your character "5").